src/main/kotlin/dev/minn/jda/ktx/interactions/builder.kt

```diff
@@ -56,6 +56,7 @@ inline fun <reified T> SubcommandData.option(name: String, description: String,
 
 fun OptionData.choice(name: String, value: String) = addChoice(name, value)
 fun OptionData.choice(name: String, value: Int) = addChoice(name, value)
```

Comment:
```suggestion
fun OptionData.choice(name: String, value: Long) = addChoice(name, value)
```
also possible if wanted / needed

Reply:
making it long is probably fine